nginx/wait-for-it.sh

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env bash
```

Comment:
Why did you switch from downloading the script to including it in our repo? Won't that make us go out of sync with improvements that are made?